src/routes/embedded/auth/recover-account/auth-recover-account.view.tsx

```diff
@@ -1,28 +1,57 @@
-import { DevFigmaScreen } from "~components/dev/figma-screen/figma-screen.component";
-
-import screenSrc from "url:/assets-beta/figma-screens/recover-account.view.png";
+import {
+  Box,
+  Button,
+  Card,
+  KeyIcon,
+  RecoverHeaderIcon,
+  Row,
+  SeedIcon,
+  Text,
+  WanderIcon
+} from "~components/embed/ui";
 
 export function AuthRecoverAccountEmbeddedView() {
   return (
-    <DevFigmaScreen
-      title="Recover your account"
-      description="After recovery, all your devices are logged out and your account recovery files are invalided. You'll have to download a new one."
-      src={screenSrc}
-      config={[
-        {
-          label: "Enter Seedphrase",
-          to: "/auth/recover-account/seedphrase"
-        },
-        {
-          label: "Import Private Key",
-          to: "/auth/recover-account/keyfile"
-        },
-        {
-          label: "Cancel",
-          to: "/auth",
-          variant: "secondary"
-        }
-      ]}
-    />
+    <Card
+      headerIcon={<RecoverHeaderIcon />}
+      headerText="Recover your account"
+      subtitle="After recovery, all your devices are logged out and your account recovery files are invalided. You'll have to download a new one."
+      footerElement={
+        <Row>
+          <Text variant={"bodyXs"} style={{ marginBottom: 0 }}>
+            {"Secured by"}
+          </Text>
+          <WanderIcon color="#838383" />
+        </Row>
+      }
+      hasBackButton={true}
+      onBackButtonClick={() => {
+        window.history.back();
+      }}
+      hasCloseButton={true}
+      onCloseButtonClick={() => {
+        window.location.href = "/auth";
```

Comment:
This will reload the app, not navigate to that route (as we use hash-based routing).